jenkins/job-configs/kubernetes-jenkins/bootstrap-ci.yaml

```diff
@@ -215,6 +215,13 @@
         frequency: 'H/5 * * * *' # At least every 30m
         json: 1
         trigger-job: 'kubernetes-build'
+    - kubernetes-e2e-gce-taint-evict:
+        job-name: ci-kubernetes-e2e-gce-taint-evict
+        jenkins-timeout: 100
```

Comment:
`self.assertGreaterEqual(job['jenkins-timeout'], job['timeout']+100, job_name)`

make this 180 and should be good to go